core/appeal/service.go

```diff
@@ -776,6 +786,7 @@ func (s *Service) handleAppealRequirements(ctx context.Context, a *domain.Appeal
 					additionalAppeal.PolicyID = aa.Policy.ID
 					additionalAppeal.PolicyVersion = uint(aa.Policy.Version)
 				}
+				ctx = context.WithValue(ctx, ContextKeyIsAdditionalAppealCreation{}, true)
```

Comment:
can we pass the flag via agument instead? 
i don't think passing flag by context is better when we could just pass it via argument.

Reply:
we can but the reason I'm using context is to keep the `appealService.Create` method clean. I don't think it should have the flag added in the param CMIIW

Reply:
@rahmatrhd passing the required value of a function via context would make the function less readable. Now the `Create` function does have 2 possible behaviour to populate policy. Adding the flag in the function argument would make it clearer what to expect.

Since the context.Value is an interface. We also will lose the `compile-type` checking benefit. 

Other alternative would be create a similar function maybe named as `Append` but this might lead to some duplicated code.

The other alternative, we could add a new variadic options in the argument and if needed user can add the flag.

I think 'clean method' does not really give benefit here if it is less readable.

Reply:
variadic options feels like a cleaner approach in this situation. Is clean and readable.

Reply:
@mabdh apart from this, can you approve this PR?